src/kouta_indeksoija_service/indexer/kouta/hakukohde.clj

```diff
@@ -30,17 +30,25 @@
   [koulutus]
   (:johtaaTutkintoon koulutus))
 
+(defn- get-alkamiskausi-koodi-uri-from-haku [haku]
+  (get-in haku [:metadata :koulutuksenAlkamiskausi :koulutuksenAlkamiskausiKoodiUri]))
+
 (defn- alkamiskausi-kevat?
   [haku hakukohde]
-  (-> (if (:kaytetaanHaunAlkamiskautta hakukohde) haku hakukohde)
-      :alkamiskausiKoodiUri
-      (clojure.string/starts-with? "kausi_k#")))
+  (let [alkamiskausi-koodi-uri (if (:kaytetaanHaunAlkamiskautta hakukohde)
+                                 (get-alkamiskausi-koodi-uri-from-haku haku)
```

Comment:
Tässä voisi olla suoraan vain `(get-in haku [:metadata :koulutuksenAlkamiskausi :koulutuksenAlkamiskausiKoodiUri])`. Toi metodi on aika lailla turha eikä sitä käytetä mistään muualta.